Doc/library/inspect.rst

```diff
@@ -838,7 +843,7 @@ function.
       :class:`Signature` objects are also supported by the generic function
       :func:`copy.replace`.
 
-   .. method:: format(*, max_width=None)
+   .. method:: format(*, max_width=None, unquote_annotations=False)
```

Comment:
This doesn't feel like a totally correct use of "unquote" to me... Maybe the parameter could be called `remove_annotation_quotes` instead of `unquote_annotation`...? I think that might also reflect slightly better the fact that not all annotations will necessarily be quoted at all.

Reply:
We use "unquote" in various other places (e.g., https://docs.python.org/3.13/library/csv.html#csv.QUOTE_NOTNULL); I feel it's more clear and concise

Reply:
"unquoted" is used in https://docs.python.org/3.13/library/csv.html#csv.QUOTE_NOTNULL as an adjective rather than a verb. I think it makes sense to talk about something being "unquoted" but less sense to talk about "unquoting" something.

Reply:
"dequote"?

Reply:
If I'm reading the code correctly, if this is false (the default) we use `repr` on annotation values that are strings, and if it's true we skip `repr` only for strings.  So this is a passive operation--if the value is true, we _don't_ do something, and when it's false we _do_ do something.  I think that's why this is awkward.

I suggest the clearest way to express this is to flip it.  Negate the boolean and rename it `quote_annotations=True`.  If it's true (the default) we quote annotations, and if it's false we don't.  I might even go with the wordier `quote_annotation_strings=True`, as we're only allowing the user to control quoting or not-quoting the annotations when they're strings.  But I don't have a strong opinion about that.

Reply:
Good idea, changing that.